docs/pages/desktop-access/active-directory-manual.mdx

```diff
@@ -1,486 +1,488 @@
 ---
-title: Desktop Access with Active Directory (Manual)
-description: Manually connect Teleport to an Active Directory domain.
+title: Configure access for Active Directory manually
+description: Explains how to manually connect Teleport to an Active Directory domain.
 videoBanner: YvMqgcq0MTQ
 ---
 
-This guide demonstrates how to connect an Active Directory domain to Teleport using the
-Desktop Service and log into a Windows desktop from that domain.
+In most cases, you can configure access for Active Directory users through the 
```

Comment:
I wouldn't say "in most cases" - in fact, I'm considering removing the scripts completely and making this the only way.